docs/settings.md

```diff
@@ -19,8 +19,8 @@ sidebar_label: Settings
 | maintenance-log-snapshotting-period | Controls time based snapshotting. New maintenancelog snapshot will be created after this period if there are new deltas | 1h | server&nbsp;only |
 | maintenance-manager-metadata-nodeset-update-period | The period of how often to check if metadata nodeset update is required | 2min | server&nbsp;only |
 | maintenance-manager-reevaluation-timeout | Timeout after which a new run is scheduled in MaintenanceManager. Used for periodic reevaluation of the state in the absence of any state changes | 2min | server&nbsp;only |
-| max-unavailable-sequencing-capacity-pct | The precentage of the sequencing capacity that is allowed to be taken down by operations, safety checker will take into account DEAD nodes as well. This means that if this value is 25, then safety checker will deny maintenances that will may take down more sequencer nodes. The percentage takes into account the weight of the sequencer nodes, so does not necessarily equals 25% of the number of sequencer nodes | 25 | server&nbsp;only |
```

Comment:
This file (`settings.md`) is actually automatically generated using: `logdeviced --markdown-settings > docs/settings.md`. Given that those fixes are in the markdown file only, the file will diverge from the generated version (and we have a test that makes sure that they are in sync which will also probably fail).
So the correct way to fix those typos is to fix the settings documentation in the cpp files and regenerate the file using the command above.

Reply:
Right, missed that. Thanks!